docs/installation.md

```diff
@@ -0,0 +1,136 @@
+# Installation
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+## Prerequisites
+
+The project has the following [dependencies](#dependencies):
+
+* [Algolia](#algolia)\*
+* [Auth0](#Auth0)
+* [Google Cloud](#google-cloud)\*
+* [Graphcool](#graphcool)\+
+* [Mailgun](#mailgun)\*
+
+> \* _optional, but recommended_
+>
+> \+ _local deployment available_
+
+## Installing
+
+First, clone the repository
+
+```bash
+# Path: ./
+git clone https://github.com/Zenika/FAQ
+```
+
+Then, install the dependencies of the project
+
+```bash
+# Path: ./FAQ/
+npm install
+```
+
+## Configuration
+
+Before running the project, you will need to configure your environment variables from the [dependencies](#dependencies)
+
+```bash
+# Path: ./FAQ/
+cp .env.local.example .env.local
+vim .env.local
+
+cp server/.env.local.example server/.env.local
+vim server/.env.local
+```
+
+## Dependencies
+
+### Algolia
+
+[Algolia](https://www.algolia.com/) is used for the full-text search.
+
+Deploy the algolia synonyms with the following command
+
+```bash
+# Path: ./FAQ/server
+npm run algolia
+```
+
+> Note: Algolia is optional for the development.
+>
+> If you do not indicate an API key for Algolia, the application will simply
+> return an empty result when searching
+
+#### Keys
+
+* App ID: _The name of your project_
+* API key (all operations) and API key (search only): _Go to "API Keys > All API keys"_
+
+### Auth0
+
+[Auth0](https://auth0.com/) is used for authentication. This is required dependency.
+
+#### Keys
+
+* Domain: _Found in your client's settings_
+* Client ID: _Found in your client's settings_
+
+### Google Cloud
+
+[Google Cloud](https://cloud.google.com/) is used to host the client on App Engine.
```

Comment:
I suggest "to host the front-end" to decrease confusion with the Auth0 client that's referred to in the previous paragraph.